### DISCUSSION TITLE
putTransportDetails Error  "CarrierName USPS is not supported"

### DISCUSSION THREAD

**Discussion by tucco78** [amzn/selling-partner-api-models#3276] [General]
Hi
We are trying to develop an fba app. While testing the putTransportDetails for Non-Partner Carriers for small parcel, we keep getting this error.  We tried all kinds of combinations USPS, UPS, United Parcel Service, etc.  
Any ideas?

Error  "CarrierName USPS is not supported"

https://github.com/amzn/selling-partner-api-models/blob/main/models/fulfillment-inbound-api-model/fulfillmentInboundV0.json#L1933

**Comment by AshwinPitroda** [comment]
@tucco78 I am also getting same error. 

Have you found any solution for this?

**Comment by tucco78** [comment]
yeah i had to dig around in their old api documentation.  I found the carrier names on these two pages for partnered and non-partnered scenarios good luck!
https://docs.developer.amazonservices.com/en_US/fba_inbound/FBAInbound_Datatypes.html#PartneredSmallParcelDataInput
https://docs.developer.amazonservices.com/en_US/fba_inbound/FBAInbound_Datatypes.html#NonPartneredSmallParcelDataInput
